mlrun/projects/pipelines.py

```diff
@@ -696,6 +696,51 @@ class _RemoteRunner(_PipelineRunner):
 
     engine = "remote"
 
+    @staticmethod
+    def _prepare_load_and_run_function(
+        source,
+        project_name,
+        save,
+        workflow_name,
+        workflow_spec,
+        artifact_path,
+        workflow_handler,
+        namespace,
+    ):
+        # Creating the load project and workflow running function:
+        load_and_run_fn = mlrun.new_function(
+            name=f"workflow-runner-{workflow_name}",
```

Comment:
constant names like these should be configured in `mlconf` as well imo.
you can set a 

```python
DEFAULT_WORKFLOW_RUNNER_NAME = "workflow-runner-{}"
```

and then use it with the `format` method of strings to set the workflow name into it.